app/src/main/java/cm/aptoide/pt/home/HomeAnalytics.java

```diff
@@ -228,14 +246,33 @@ public void sendAppsChipHomeInteractEvent() {
   public void sendAppsChipInteractEvent() {
     final Map<String, Object> data = new HashMap<>();
     data.put(ACTION, APPS);
-    analyticsManager.logEvent(data, HOME_CHIP_CLICK, AnalyticsManager.Action.CLICK,
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
```

Comment:
Apps and Games interact event could become one with the the chip type being passed as a parameter.